.github/workflows/test.yml

```diff
@@ -10,39 +10,19 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-        - '3.8'
```

Comment:
PS: I had to rebase this so make sure to do a pull.

Reply:
Django 5.x supports Python 3.10 and above, so I refactored CI config to use include instead of exclude, so Django 4.2 will be using Python 3.8 and 3.9. Otherwise we have to exclude 3.8 and 3.9 from all 5.x versions, which results more config lines.